python_codes/contour_plots.py

```diff
@@ -43,13 +43,18 @@
 # NH3 Variations
 
 NH3avg = mean(NH3_total, axis = (0,2))
-print(NH3_total.shape, NH3avg.shape)
 NH3_variations = NH3_total - NH3avg.reshape((1,nx1,1))
```

Comment:
Let's formalize the naming convention. 
Temperature: **temp**
Pressure: **pres**
NH3 vapor mass mixing ratio: **qNH3v**
NH3 vapor density: **rhoNH3v**
NH3 cloud density: **rhoNH3c**
NH3 cloud mass mixing ratio: **qNH3c**
NH3 precipitation mass mixing ratio: **qNH3p**
H2O total mass mixing ratio: **qH2Ot**
temporal average: **_bar**
horizontal average: **_avg**
anomaly with respect to some reference value: **a**
so **NH3_variations** should be changed to **qNH3a**